translations/ui-invoice/en.json

```diff
@@ -238,6 +238,15 @@
   "invoice.lockTotalAmount.tooltip": "Check Lock total to interact with this field",
   "invoice.validation.accountingCode": "To export to accounting you must select an accounting code",
 
+  "filter.createdBy": "Created by",
+  "filter.updatedBy": "Updated by",
+  "filter.dateCreated": "Date created",
+  "filter.dateUpdated": "Date updated",
+  "filter.invoiceLine.createdBy": "Created by (Invoice Line)",
```

Comment:
I don't see such requirements in the ticket, are you sure you need to do anything for invoice lines?